subprojects/ide/src/main/java/org/gradle/plugins/ide/idea/internal/IdeaScalaConfigurer.java

```diff
@@ -138,6 +138,9 @@ private static ProjectLibrary createScalaSdkLibrary(Project scalaProject, Iterab
         if (runtime != null) {
             FileCollection scalaClasspath = runtime.inferScalaClasspath(files);
             File compilerJar = runtime.findScalaJar(scalaClasspath, "compiler");
+            if (compilerJar == null) {
```

Comment:
Should there be a notion of priority there?
Like ignore `compiler` if `compiler_3` is present?

Reply:
There shouldn't be both on the path (and we define it ourselves here), but I could switch it around.

Reply:
I left it as is for now, so if there is a conflict on the classpath, we choose Scala 2 (in reality this should never happen unless something goes very wrong)